### ISSUE TITLE
pg_search_documents is missing index on searchable_id and searchable_type

### ISSUE THREAD

**Issue by listenloop** [Casecommons/pg_search#219]
pg_seach_documents table is added by the rake task `CreatePgSearchDocuments'. It setups up the polymorphic relationship but the columns are not indexed.

featching a record from PgSearch does a sequence scan as there is no index on it. This is too slow. 

Everytime a model with has multi_searchable association is updated, it does a sequence scan on the pg_search_documents table which could be very slow depending upon the size of the DB.

#With No index
`PgSearch::Document.where("searchable_id = 499214 and searchable_type = 'Visitor'").explain`

EXPLAIN for: SELECT \"pg_search_documents\".* FROM \"pg_search_documents\"  WHERE (searchable_id = 499214 and searchable_type = 'Visitor')\n                                      QUERY PLAN\n--------------------------------------------------------------------------------------\n Seq Scan on pg_search_documents  (cost=0.00..40298.58 rows=1 width=89)\n   Filter: ((searchable_id = 499214) AND ((searchable_type)::text = 'Visitor'::text))\n(2 rows)\n

#With index
Index Scan using index_pg_search_documents_on_searchable_id_and_searchable_type on pg_search_documents  (cost=0.42..8.45 rows=1 width=89)\n   Index Cond: ((searchable_id = 499214) AND ((searchable_type)::text = 'Visitor'::text))\n(2 rows)

Fix
```ruby
def up
    add_index :pg_search_documents, [:searchable_id, :searchable_type]
  end

  def down
    remove_index :pg_search_documents, [:searchable_id, :searchable_type]
  end
```

**Comment by listenloop** [comment]
I have submitted a pull request for this: https://github.com/Casecommons/pg_search/pull/220

**Comment by nertzy** [comment]
Fixed by 86ee609ff7a780c62a7d3d83c781a67d4b3c168d
